src/slic3r/GUI/UserAccountCommunication.hpp

```diff
@@ -12,6 +12,8 @@
 #include <mutex>
 #include <memory>
 
+#include <wx/timer.h>
```

Comment:
Maybe you want to forward declare the referenced classes here only by:
```
class wxTimer;
class wxTimerEvent;
```
Then move this include into the cpp file above to keep the scope minimal.